packages/block/test/block.spec.ts

```diff
@@ -6,16 +6,16 @@ import { Block } from '../src'
 tape('[Block]: block functions', function (t) {
   t.test('should test block initialization', function (st) {
     const common = new Common({ chain: 'ropsten', hardfork: 'chainstart' })
-    const block1 = Block.fromBlockData({}, { common: common, initWithGenesisHeader: true })
-    st.ok(block1.hash().toString('hex'), 'block should initialize')
+    const genesis = Block.genesis({}, { common })
```

Comment:
Ah, on the other hand (on the signature question): this also looks pretty simple already, so maybe not worth the further simplification, not sure.